app/src/main/java/com/nextcloud/talk/controllers/ConversationInfoController.kt

```diff
@@ -684,4 +684,23 @@ class ConversationInfoController(args: Bundle) : BaseController(args), FlexibleA
 
         private const val ID_DELETE_CONVERSATION_DIALOG = 0
     }
+
+    /**
+     * Comparator for participants, sorts by online-status, moderator-status and display name.
```

Comment:
> Sorts the participants list in a hierarchical way: ownership, online, moderator, displayname (like the webUI)

Web sorts:
* _Later:_ actor type (only with api v4) =>participants with actor type group last
* participant has a session (_Later:_ user status === offline forces false) => participants with sessions first
* participant in call flag is !== 0 => participant with !== 0 first
* can moderate (check participant type [OWNER, MODERATOR, GUEST_MODERATOR]) => moderator powers first
* _Later:_ user status is dnd => dnd last
* Display name => alphabetically case-insensitive

https://github.com/nextcloud/spreed/blob/master/src/components/RightSidebar/Participants/CurrentParticipants/CurrentParticipants.vue#L139-L201

Reply:
Most of the infos aren't available on the Android client yet :/

So I changed the sorting to 
* online status
* moderator type(s) (moderator, owner; guest_moderator is not known to the client yet or rather not processed)
* Display name => alphabetically case-insensitive